hail/python/hail/context.py

```diff
@@ -398,3 +398,7 @@ def _set_flags(**flags):
     if len(invalid) != 0:
         raise FatalError("Flags {} not valid. Valid flags: \n    {}"
                          .format(', '.join(invalid), '\n    '.join(available)))
+
+
+def _get_flags(*flags):
+    return {flag: Env.hc()._jhc.flags().get(flag) for flag in flags}
```

Comment:
I thought we weren't using the java hail context anymore.

Reply:
This is for feature flags for the spark backend, I don’t know if there’s a second way but I can look.

Reply:
Doesn't look like there's an endpoint on the service backend that would allow setting of feature flags. I added a decorator to the test method to make sure it's not run on service tests. It wouldn't be hard to add feature flags to the service backend but I'm not sure if that's something we want to do now.